### PR TITLE
Add net.sf.VICE

### PR DESCRIPTION
**Pull request by digitalstreamio**

### CODE REVIEW

File: net.sf.VICE.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">net.sf.VICE.desktop</id>
+  <name>VICE</name>
+  <summary>Versatile Commodore Emulator</summary>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
```
Comment (barthalion) [inline net.sf.VICE.appdata.xml]: If you are the author of this file, we prefer `CC0-1.0`.

File: net.sf.VICE.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    </p>
+  </description>
+  <url type="homepage">http://vice-emu.sourceforge.net/</url>
+  <url type="bugtracker">https://sourceforge.net/p/vice-emu/bugs/</url>
+  <screenshots>
+    <screenshot type="default">https://raw.githubusercontent.com/digitalstreamio/flatpak-images/master/net.sf.VICE/screenshots/1.png</screenshot>
+  </screenshots>
+  <releases>
+    <release date="2018-02-24" version="3.1"/>
+  </releases>
```
Comment (barthalion) [inline net.sf.VICE.appdata.xml]: Please add [OARS](https://hughsie.github.io/oars/) ratings.

File: net.sf.VICE.json [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    "command": "x64",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "arch": {
+            "x86_64": {
+                "cflags": "-O3 -g -pipe -Wall -m64 -march=nocona -mtune=core2 -fexceptions -ffast-math -fstack-protector --param=ssp-buffer-size=4 -Wp,-D_FORTIFY_SOURCE=2"
```
Comment (barthalion) [inline net.sf.VICE.json]: Does it really have to be that specific? Also what's the point of `-Wall`?

File: net.sf.VICE.desktop [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=X64 (Vice)
```
Comment (barthalion) [inline net.sf.VICE.desktop]: Is it commonly used name for C64 emulator? (Honest question, for a layman it would be confusing.)

**Comment by barthalion** [comment]
bot, build net.sf.VICE

**Comment by barthalion** [comment]
Build status if it doesn't get reported automatically: https://flathub.org/builds/#/builders/1/builds/2270

File: net.sf.VICE.appdata.xml (hunk 1 above)
Comment (digitalstreamio) [reply net.sf.VICE.appdata.xml]: Done.

File: net.sf.VICE.appdata.xml (hunk 2 above)
Comment (digitalstreamio) [reply net.sf.VICE.appdata.xml]: Done

File: net.sf.VICE.desktop (hunk 4 above)
Comment (digitalstreamio) [reply net.sf.VICE.desktop]: Good catch. I changed the name to "VICE (C64)" to match the app and the specific emulator mode.

File: net.sf.VICE.json (hunk 3 above)
Comment (digitalstreamio) [reply net.sf.VICE.json]: How about these?

"cflags": "-O3 -g -m64 -march=nocona -mtune=core2 -ffast-math"

The selected options try to extract most out of current gen cpus while remaining compatible with 1st gen 64-bit cpus (Pentium 4/Athlon 64). However, I don't have any numbers to measure the speed difference so if you feel strongly about this, I can drop it to defaults.

**Comment by nedrichards** [comment]
bot, build net.sf.VICE

File: net.sf.VICE.json (hunk 3 above)
Comment (barthalion) [reply net.sf.VICE.json]: I'd rather keep it generic (and maybe switch to `-Ofast` for -zomg-optimize), but it can wait until the merge.

**Comment by barthalion** [comment]
Marking as ready, first i386 build failed due to SF issue, retrigger went well: https://flathub.org/builds/#/builders/56/builds/8

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/net.sf.VICE

Make sure to accept the invite to collaboration. (You will receive two as I invited you to wrong one by mistake, so please just ignore it.)

Thanks!

**Comment by digitalstreamio** [comment]
Awesome. Thanks for all your help!

**Pull request closed without merging**